File: pset6/mario.py
import cs50

def main():
    while True:
        print("Height: ",end="")
        n=cs50.get_int()
        if n>0 and n<=23:
            break
    space = n-1
    for i in range(n):
        print(" "*space + "#"*(i+2))
        space-=1


if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

void spaces(int n);
void hashes(int n);

int main(void)
{
    int n;
    do
    {
        printf("Height: ");
        n = get_int();
    }
    while((n<0)||(n>23));
    int j=2;
    for(int i=n-1; i>=0; i--)
    {
        spaces(i);
        hashes(j);
        j++;
        printf("\n");
    }
}

void spaces(int n)
{
    for(int i=0;i<n;i++)
    {
        printf(" ");
    }
}

void hashes(int n)
{
    for(int i=0;i<n;i++)
    {
        printf("#");
    }
}